src/Constants.php

```diff
@@ -46,13 +67,20 @@ public static function init($dir)
 	private static function env_set($key, $default)
 	{
 		// Check if we have env value
-		$env_value = env($key, $default);
+		$env_value = env($key);
 
-		if (! isset($env_value)) {
-			return;
-		}
+        // If not value or default move along
+        if (!isset($env_value) && !isset($default)){
+            return;
+        }
+
+        // Use the default if env_value is not set
+        if(!isset($env_value)){
+            DotEnv::set($key,$default);
```

Comment:
This ensures the default $env_vars values are available via the env() helper method.